pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -156,9 +156,8 @@ func (cache *claimInfoCache) withRLock(f func() error) error {
 }
 
 // add adds a new claim info object into the claim info cache.
-func (cache *claimInfoCache) add(info *ClaimInfo) *ClaimInfo {
+func (cache *claimInfoCache) add(info *ClaimInfo) {
 	cache.claimInfo[info.Namespace+"/"+info.ClaimName] = info
-	return info
```

Comment:
The semantic of returning the ClaimInfo instance was unclear (is it the same or potentially a different instance?), so to avoid the ambiguity I removed this.